src/Symfony/Bridge/Twig/Extension/WorkflowExtension.php

```diff
@@ -21,6 +21,7 @@
  * WorkflowExtension.
  *
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Carlos Pereira De Amorim <carlos@shauri.fr>
```

Comment:
I added this because I made a modification in a previous request and I forgot to add it.
Sorry